frontend/src/components/requisitionForm/RequisitionItemCard.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { Button, Card, Col, Form, Input, InputNumber, Row, Tooltip } from "antd";
+import { QuestionCircleOutlined } from "@ant-design/icons/lib";
+
+const { TextArea } = Input;
+
+interface FieldData {
+  name: number;
+  key: number;
+  fieldKey: number;
+}
+
+interface Props {
+  field: FieldData;
+  deleteButton: boolean;
+  rules: any;
+  remove: (index: number) => void;
+}
+
+const RequisitionItemCard: React.FC<Props> = (props) => (
+  <Card
+    size="small"
+    title={`Item ${props.field.name + 1}`}
+    style={{ marginBottom: "15px" }}
+    extra={props.deleteButton
+      ? (
+        <Button onClick={() => props.remove(props.field.name)} type="text" size="small" danger>
+          Delete
+        </Button>
+      ) : null}
+  >
+    <Row gutter={[8, 0]}>
+      <Col xs={24} sm={12}>
+        <Form.Item
+          name={[props.field.name, "name"]}
+          fieldKey={[props.field.fieldKey, "name"]}
+          rules={[props.rules.requiredRule]}
+          label="Name"
+        >
+          <Input placeholder="Sparkly crayons" />
+        </Form.Item>
+      </Col>
+
+      <Col xs={24} sm={12}>
+        <Form.Item
+          name={[props.field.name, "link"]}
+          fieldKey={[props.field.fieldKey, "link"]}
+          rules={[props.rules.requiredRule, props.rules.urlRule]}
+          label={(
+            <span>
+              {"Link "}
+              <Tooltip title="The link to the item you want to purchase">
+                <QuestionCircleOutlined />
+              </Tooltip>
+            </span>
+          )}
+        >
+          <Input type="url" placeholder="http://rubberducks.com" />
+        </Form.Item>
+      </Col>
+    </Row>
+
+    <Row gutter={[8, 0]}>
+      <Col xs={24} sm={12}>
+        <Form.Item
+          name={[props.field.name, "quantity"]}
+          fieldKey={[props.field.fieldKey, "quantity"]}
+          rules={[props.rules.requiredRule]}
+          label="Quantity"
+        >
+          <InputNumber
+            prefix="$"
+            type="number"
+            min={1}
+            precision={0}
+            style={{ width: "100%" }}
+            placeholder="578"
+          />
```

Comment:
- prefix shouldn't be $ if this is quantity
- precision of 0 seems to mean any decimal value entered gets rounded/truncated to 0 decimal places.  Even though this is a quantity input, I'm not sure this is entirely desirable because partial quantities are sometimes a thing

Reply:
I took out the dollar. Yes, that is the functionality. Currently, the db is setup to only accept positive integers. Should I go ahead and change everything to decimals?

Reply:
Actually, I looked at our spring purchase requests and there were 0 with a decimal quantity.  So I think a positive integer is fine for that